alf/algorithms/actor_critic_loss.py

```diff
@@ -28,15 +28,33 @@
                                  ["pg_loss", "td_loss", "neg_entropy"])
 
 
+def normalize(batch_norm, x):
+    batch_norm.train()
+    momentum = batch_norm.momentum
+    if batch_norm.num_batches_tracked * momentum < 1.0:
+        # First the first few batches, we do cumulative moving average
+        batch_norm.momentum = None
+    batch_norm(x)
+    batch_norm.momentum = momentum
+    # We use the running mean and variance of the advantages to normalize
+    # since the batch may not be large enough to properly normalize within
+    # the batch.
+    batch_norm.eval()
+    return batch_norm(x)
+
+
 @alf.configurable
 class ActorCriticLoss(Loss):
     def __init__(self,
+                 reward_dim=1,
```

Comment:
Need to update the docstring for `reward_dim`.

Reply:
updated.